Engine/Code/Engine/Graphics/Renderer.cpp

```diff
@@ -327,20 +353,27 @@ void Renderer::DrawQuad(const Vec2& position, Vec2 meshDim, Vec4 color, const st
 	delete mb;
 }
 
-
 //------------------------------------------------------------------------------------------------------
 // Drawing a FramBuffer
 
 void Renderer::DrawFrameBuffer(const Vec2& position, Vec2 meshDim)
```

Comment:
I thought I told you before that this function is not named right? You can't draw a framebuffer .. What you're doing here is DrawMeshImmediate

Reply:
Or if I'm ready currect its drawing a full screen quad .. So the name would be DrawFullScreenQuad

Reply:
Done